src/ui/public/agg_types/metrics/latest.js

```diff
@@ -0,0 +1,44 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import orderAndSizeTemplate from 'ui/agg_types/controls/order_and_size.html';
+
+export default function AggTypeMetricLatestProvider(Private) {
+  let MetricAggType = Private(AggTypesMetricsMetricAggTypeProvider);
+
+  return new MetricAggType({
+    name: 'top_hits',
+    title: 'Latest',
+    makeLabel: function (aggConfig) {
```

Comment:
all of this is based on the other metrics types, if you check any other file in this folder (like sum.js, max.js, ...), you will see they look the same i didnt change much.

would it be a good idea to update all of them to follow the new style ?  (this goes for all the comments on this file, i wont be repeating it in the others) as we probably dont want every metric type to have its own style .... they do similar thing, they should look the same.